### DISCUSSION TITLE
Docker Compose ATM8 - Only installs version 1.0.10

### DISCUSSION THREAD

**Discussion by Bcaruso22** [itzg/docker-minecraft-server#1985] [Q&A]
`version: "3"

services:
  minecraft:
    image: itzg/minecraft-server
    volumes:
      - mcdataatm:/data
      - /home/AllTheMods8/mods:/mods
    ports:
      - 25600:25565
    environment:
      EULA: "TRUE"
      TYPE: "AUTO_CURSEFORGE"
      CF_SLUG: "all-the-mods-8"
      CF_FILE_ID: "4415465"
    tty: true
    stdin_open: true
    restart: unless-stopped
volumes:
  mcdataatm:
    external: true`

That's my yml file. I've tried using:
CF_PAGE_URL: https://www.curseforge.com/minecraft/modpacks/all-the-mods-8/files/4415465

CF_SLUG: "all-the-mods-8"
CF_FILE_ID: "4415465"

CF_SLUG: "all-the-mods-8"
CF_FILENAME_MATCHER: "1.0.13"

None of them would download version 1.0.13. Every time it downloads version 1.0.10.

I've have use compose to setup another server for my kids that has a custom mod pack i created and it worked great. I'm not sure what I am doing wrong. Any help would be appreciated

**Comment by itzg** [comment]
Try pulling the latest image because I recently fixed an issue where the CurseForge listing API was caching results.

**Comment by Bcaruso22** [comment]
I tried again, but same issue. I imagine I was doing something wrong.
I just deleted the volume and container and started over. Worked perfectly.

The only thing I noticed was first attempt I did not specify a version and let it install newest version, but it was only installing .10 as I said. The second time around I used the slug & file ID for .13 and it worked.

I kind of wish I had used the same method as the first to see if it recognized the .13 as the newest and installed it. However I got what I wanted. I'll try changing my .yml file to install most recent after a few updates and see what it does.

Worse case, I save my world folder and recreate it all. Doesn't take long.

Thank you for the quick reply last yesterday.

**Reply by itzg** [reply]
I'm glad you were able to make progress. If you run into it again, set `DEBUG` to "true" and provide those logs.
